### PR TITLE
Update cargo-winrt output to better match cargo's style

### PR DESCRIPTION
**Pull request by zachlute**
Add print_status and print_verbose_status methods with consistent justified formatting.

As specified in #247 .

cargo_winrt_output

### CODE REVIEW

File: crates/cargo-winrt/src/main.rs [hunk 1]
```diff
@@ -215,10 +237,10 @@ impl Install {
...
                 .collect::<anyhow::Result<Vec<DependencyDescriptor>>>()?
         };
-        debug!(
-            "{} {} nuget dependencies",
-            console::style("Installing").green().bold(),
-            dependency_descriptors.len()
+
+        print_verbose_status(
+            "Installing",
+            format_args!("{} nuget dependencies", dependency_descriptors.len()),
```
Comment (zachlute) [inline crates/cargo-winrt/src/main.rs:243]: The `format_args!` are *probably* overkill, but the previous `debug!` macro for verbose output wouldn't evaluate the formatted string unless `-v` was actually specified, so I wanted to at least prove that this change could maintain that behavior if desired.

File: crates/cargo-winrt/src/main.rs [hunk 2]
```diff
@@ -169,32 +170,53 @@ fn verbose() -> bool {
...
+    let secs = duration.as_secs();
+
+    if secs >= 60 {
+        format!("{}m {:02}s", secs / 60, secs % 60)
+    } else {
+        format!("{}.{:02}s", secs, duration.subsec_nanos() / 10_000_000)
+    }
+}
+
+fn print_status<D>(status: &str, message: D)
```
Comment (rylev) [inline crates/cargo-winrt/src/main.rs:184]: I think I would prefer these as macros so you're not required to do the funny `format_args` or `format` dance. Is there a reason to not use macros for this?

Comment (zachlute) [reply crates/cargo-winrt/src/main.rs:184]: I was mostly borrowing from the way Cargo does this output, while maintaining the semantics already used here, so it wasn't explicitly a decision to 'not use macros'. Happy to give it a shot with macros if you think that'd be more useful.

**Comment by kennykerr** [comment]
Thanks @zachlute - works great!

**Pull request merged**